src/net/sourceforge/kolmafia/swingui/widget/RequestPane.java

```diff
@@ -1,25 +1,123 @@
 package net.sourceforge.kolmafia.swingui.widget;
 
+import java.awt.*;
 import java.io.StringWriter;
+import java.util.Enumeration;
 import java.util.regex.Pattern;
-import javax.swing.JEditorPane;
+import javax.swing.*;
+import javax.swing.text.*;
+import javax.swing.text.html.HTML;
 import javax.swing.text.html.HTMLDocument;
+import javax.swing.text.html.HTMLEditorKit;
 import net.sourceforge.kolmafia.KoLConstants;
 import net.sourceforge.kolmafia.preferences.Preferences;
 import net.sourceforge.kolmafia.utilities.StringUtilities;
 
 public class RequestPane extends JEditorPane {
+  static class WrappedHtmlEditorKit extends HTMLEditorKit {
+    private final ViewFactory viewFactory;
+
+    public WrappedHtmlEditorKit() {
+      super();
+      this.viewFactory = new WrappedHtmlFactory();
+    }
+
+    @Override
+    public ViewFactory getViewFactory() {
+      return this.viewFactory;
+    }
+
+    private class WrappedHtmlFactory extends HTMLEditorKit.HTMLFactory {
+      @Override
+      public View create(Element elem) {
+        View view = super.create(elem);
+
+        if (view instanceof LabelView) {
+          Object attribute = elem.getAttributes().getAttribute(StyleConstants.NameAttribute);
+
+          if ((attribute instanceof HTML.Tag) && (attribute == HTML.Tag.BR)) {
+            return view;
+          }
+
+          return new WrapLabelView(elem);
+        }
+
+        return view;
+      }
+
+      private class WrapLabelView extends LabelView {
+        private String title;
+
+        public WrapLabelView(Element elem) {
+          super(elem);
+
+          Enumeration<?> itel = elem.getAttributes().getAttributeNames();
+
```

Comment:
What is "itel" supposed to suggest to the reader?

Reply:
Renamed to iterator. Old habit of mine.